cgi/test_ingredients_analysis.pl

```diff
@@ -104,6 +96,9 @@
 	$full_width = 0;
 }
 
+$tt->process('test_ingredients_analysis.tt.html', $template_data_ref, \$html);
```

Comment:
It's best to use this new function:

process_template('test_ingredients_analysis.tt.html', $template_data_ref, \$html) or $html = "<p>" . $tt->error() . "</p>";

Reply:
That function probably was not in the script you used as an example, we'll need to change the old scripts as well so that they use it.

Reply:
yes, it was in `spellcheck_test.pl`, should I update that file too?

Reply:
sure you can do it, in a new PR maybe